src/gui/ProcessorEditor.cpp

```diff
@@ -40,9 +42,10 @@ ProcessorEditor::ProcessorEditor (BaseProcessor& baseProc, ProcessorChain& procs
     infoSvg->replaceColour (Colours::black, contrastColour);
     infoButton.setImages (infoSvg.get());
     addAndMakeVisible (infoButton);
-    infoButton.onClick = [&baseProc, boardComp = dynamic_cast<BoardComponent*> (parent)]
+    infoButton.onClick = [&baseProc, &parent]
     {
-        boardComp->showInfoComp (baseProc);
+        if (auto* boardComp = dynamic_cast<BoardComponent*> (parent))
+            boardComp->showInfoComp (baseProc);
```

Comment:
Can we revert this? Ideally we wouldn't have to `dynamic_cast` on every click...